src/firebase/firebase.js

```diff
@@ -0,0 +1,17 @@
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+const config = {
+  apiKey: 'AIzaSyDdyVFa9xk5ts63G2ST4Xtm2kHWrku3z4U',
```

Comment:
Don't think we should be committing keys like this going forward (especially since I think that frontend files like this would be made available in the client's browser, can anyone confirm this?).

Realistically the firebase configs would be better stored in a backend and React wouldn't need to know anything about firebase at all and would just get data it needs by requesting from the backend.

Reply:
I was trying to get a working model going for the project. I do agree that it should be stored away from the front-end.

Reply:
Built a production build using `npm run build` and served it on a static server using `serve -s build`. It is possible to access the configs file from dev tools. So this is a pretty big security concern atm.

Reply:
I started looking into this and apparently everything in the config file is okay to leave in a public file like this. What's important is setting up firebase's security rules correctly to prevent unwanted users from reading or writing data. This makes sense as Google even puts this config file in its documentation [here](https://firebase.google.com/docs/web/setup)
For more info I found this stackoverflow question [here](https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public)